packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -89,7 +94,15 @@ export const NotifiSubscriptionContextProvider: React.FC<
   const [email, setEmail] = useState<string>('');
   const [phoneNumber, setPhoneNumber] = useState<string>('');
   const [telegramId, setTelegramId] = useState<string>('');
-  const [discordId, setDiscordId] = useState<string>('');
+
+  const [discordTargetData, setDiscordTargetData] = useState<DiscordTarget>({
```

Comment:
probably cleaner if we do `useState<DiscordTarget | undefined>(undefined)` instead of having a "fake" unconfirmed target like this

Reply:
! done